server/src/api/resolvers/mutation.ts

```diff
@@ -215,6 +215,18 @@ const updateVendor = async function updateVendor(parent: any, args: MutationUpda
     });
 }
 
+const updateBudget = async function updateBudget(parent: any, args: MutationUpdateBudgetArgs) {
+    return await prisma.budget.update({
+        where: {
+            id: args.id
+        },
+        data: {
+            ...args.data,
+            name: args.data.name
```

Comment:
You can delete line 225, since that will already be included by the ... spread operator above

Reply:
u right